alibi_detect/cd/tests/test_spot_the_diff.py

```diff
@@ -57,7 +57,7 @@ def test_stddrift(stddrift_params):
         kernel = MyKernelTF((n_features,))
     else:
         kernel = None
-    x_ref = np.random.randn(*(n, n_features))
+    x_ref = np.random.randn(*(n, n_features)).astype(np.float32)
```

Comment:
`x_ref` was not previously used during detector instantiation, but now is used to build the model internally. The model expects float32 data...